cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -82,3 +83,43 @@ def is_ansible_pod_stopped(enabled_embedded_appliance):
         assert wait_for(is_nginx_stopped, func_args=[enabled_embedded_appliance], num_sec=30)
     else:
         assert wait_for(is_ansible_pod_stopped, func_args=[enabled_embedded_appliance], num_sec=180)
+
+
+@pytest.mark.tier(1)
+def test_embed_tower_event_catcher_process(appliance, enabled_embedded_appliance):
```

Comment:
Don't you need to add `test_requirements.ansible` here? I see it in the manual file, but not here.

Reply:
Updated. Thanks for quick review.

Reply:
I appreciate the quick update :smile:

Reply:
@sbulage You will need to import `test_requirements` here. 
`from cfme import test_requirements`

Reply:
Updated.